src/views/wishlist.pug

```diff
@@ -16,6 +16,13 @@ block title
               i.fas.fa-sync-alt
         .level-item
           span= lang('WISHLIST_TITLE', name)
+      .level-right
+        .level-item 
+          a(href="#addWishlistItem").button.button.is-primary.is-rounded
+            span.icon
+              i.fas.fa-plus
+            |&nbsp;&nbsp;Add Item
```

Comment:
This should be localized with =lang() and should use padding rather than &nbsp;.

Reply:
Yep, good spot. Added with https://github.com/Wingysam/Christmas-Community/pull/148/commits/ebc0c4fcb6222ba9b334d24ea219977b2d87698f